mitmproxy/contrib/kaitaistruct/exif.py

```diff
@@ -0,0 +1,573 @@
+# This is a generated file! Please edit source .ksy file and use kaitai-struct-compiler to rebuild
+
+import array
+import struct
+import zlib
+from enum import Enum
+
+from kaitaistruct import KaitaiStruct, KaitaiStream, BytesIO
+
+
+class Exif(KaitaiStruct):
```

Comment:
The only variables I get from the `app1` segment are the ones in the `__init__`. The `exif` data is never parsed.